src/codeu/chat/server/contexts/UserContext.java

```diff
@@ -69,4 +70,21 @@ public String removeConvoInterest(String name) {
     return super.removeConvoInterest(name);
   }
 
+  @Override
+  public void deleteConversation(Uuid conversationId) throws SecurityViolationException {
+    ConversationContext foundConversation = null;
+    for (ConversationContext convo : conversations()) {
+      if (convo.conversation.id.equals(conversationId)) {
+        foundConversation = convo;
+      }
+    }
+    // HACK: I want to throw an IllegalArgumentException but without changing the method signature
+    if (foundConversation == null) return;
```

Comment:
IMO it makes sense to throw security violation even if it doesn't exist (but it's fine if it doesn't)